client/src/files.rs

```diff
@@ -111,7 +111,9 @@ impl FileLocations {
 
     /// Return default path for database, depending on network ID
     pub fn database(&self, network: NetworkId) -> PathBuf {
-        self.database_parent.join(format!("db.{}", network))
+        // NOTE: This is analog to the names in core-js. But since we currently only have full
+        // consensus, this part is hard-coded.
+        self.database_parent.join(format!("{}-full-consensus", network))
```

Comment:
Macro sync still needs integration into the client API anyway. It's really high on my ToDo list. I will do this in a seperate merge request.

Reply:
Ah yes, also this will be replaces soon. `nimiq-lib2` has a `StorageConfig` which abstracts the storage backend and paves the way to use in-browser storage. Or actually any storage you want. For all I care, you can store your blockchain in IPFS or a twitter feed :man_shrugging:

But this doesn't need changes as it will become obsolete. Furthermore the `StorageConfig` already considers the consensus/sync mod:

```rust
let db_name = format!("{}-{}-consensus", network_id, consensus).to_lowercase();
```